shell/platform/darwin/ios/framework/Headers/FlutterBinaryMessenger.h

```diff
@@ -51,6 +51,9 @@ FLUTTER_EXPORT
  */
 - (void)sendOnChannel:(NSString*)channel message:(NSData* _Nullable)message;
 
+// TODO: Add macOS support for replies once
+// https://github.com/flutter/flutter/issues/18852 is fixed.
+#if TARGET_OS_IPHONE || defined(FLUTTER_TEST)
```

Comment:
Please use the `API_AVAILABLE` and related macros from `os/availability.h` (here and elsewhere) instead of adding guards.

Reply:
That's much cleaner, thanks!